package.json

```diff
@@ -66,5 +67,8 @@
         "@types/web": "npm:typescript@~4.7.4",
         "eslint": "8.22.0",
         "@ngraveio/bc-ur": "1.1.12"
+    },
+    "@parcel/resolver-default": {
+        "packageExports": true
```

Comment:
Can you explain what this does?

Reply:
CC @tomiir

Reply:
Hi @steveluscher ! My bad for not including it in the description (editing right now). 

Seems like Parcel v2.9+ requires this to be set on the root package json in order to properly parse path exports from other packages. 

Here's the related blog post from the parcel team: https://parceljs.org/blog/v2-9-0/#new-resolver